LandLord-core/src/main/java/biz/princeps/lib/command/SubCommand.java

```diff
@@ -19,7 +19,7 @@ public abstract class SubCommand implements Command {
     private final String usage;
     private final Set<String> aliases;
 
-    public SubCommand(String name, String usage, Set<String> permissions, Set<String> aliases) {
+    protected SubCommand(String name, String usage, Set<String> permissions, Set<String> aliases) {
```

Comment:
As this targets v5, breaking changes are good to go.
However, Landlord lib things may stay accessible/public imo, waiting for whole team view on this.

Reply:
This is not breaking because you can only call the constructor of an abstract class if you extend it.